intelmq/bots/BOTS

```diff
@@ -1034,6 +1034,7 @@
                     "",
                     "source.fqdn"
                 ],
+                "headers": true,
```

Comment:
I'd prefer if the syntax was very clear what this does.
How about renameing to "skip_header": true  ?

Otherwise, I'll merge that in of course.
Nice addition, thanks!